### PR TITLE
XIVY-6925 praise for the 'role based welcome' screen

### PR DESCRIPTION
**Pull request by ivy-rew**

### CODE REVIEW

**Comment by ivy-rew** [comment]
newsRoleBased

**Comment by ivy-rew** [comment]
you may browse the site here: http://192.168.3.52/news/9.3#role93

File: src/app/pages/news/9.3/04-role-welcome.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+- **Project filters**: contents within the Project tree will be filtered according to your role. Either to reduce complexity or to expose advanced configs. You can switch at any time to a different filterset.
 
 <div class="short-links">
-	<a href="/portal/9.2/doc/portal-developer-guide/introduction/index.html#new-noteworthy-9-2"
+	<a href="${docBaseUrl}/designer-guide/process-modeling/projects/ivyProjectView.html"
 		target="_blank" rel="noopener noreferrer">
-		<i class="si si-book"></i> Portal New & Noteworthy
+		<i class="si si-book"></i> Designer Guide
 	</a>
 </div>
```
Comment (ivy-bb) [inline src/app/pages/news/9.3/04-role-welcome.md:14]: Top! but I would show the xNewWelcomePage.png as first "big" image

Comment (ivy-rew) [reply src/app/pages/news/9.3/04-role-welcome.md:14]: Thanks @ivy-bb , I've adjust the the picture order as suggested

**Pull request merged**